test/test.js

```diff
@@ -42,6 +42,7 @@
   var phantom = root.phantom,
       amd = root.define && define.amd,
       argv = root.process && process.argv,
+      jsArrayFrom = Array.from,
```

Comment:
I thought it'd be better to ensure it conforms to an actual browser implementation